src/core/network.js

```diff
@@ -75,6 +75,21 @@ var NetworkManager = (function NetworkManagerClosure() {
     return array.buffer;
   }
 
+//#if (GENERIC || FIREFOX || B2G || MOZCENTRAL)
```

Comment:
I'm just wondering: Since we know that `moz-chunked-arraybuffer` is supported for `MOZCENTRAL` (and `FIREFOX` too, for all versions that we realistically care about), couldn't we avoid doing any additional checks for those cases? E.g. something like this:
```javaScript
//#if (FIREFOX || MOZCENTRAL)
//  var supportsMozChunked = true;
//#else
    var supportsMozChunked = (function supportsMozChunkedClosure() {
      var x = new XMLHttpRequest();
      .
      .
      .
//#endif
```